lambdas/checkpassport/src/test/java/uk/gov/di/ipv/cri/passport/checkpassport/services/DocumentDataVerificationServiceTest.java

```diff
@@ -114,7 +122,7 @@ void verifyIdentityShouldReturnResultWhenValidInputProvided(boolean documentVeri
         assertEquals(documentVerified, documentDataVerificationResult.isVerified());
         assertEquals(documentVerified ? 2 : 0, documentDataVerificationResult.getValidityScore());
         assertEquals(
-                documentVerified ? 0 : 1,
+                documentVerified ? 0 : 2,
```

Comment:
Why do we have more CIs than before?

Reply:
The mockContraIndicatorMapper response was mocked which has test flag with a new test CI1. The test flag and the standard response give the new list size here. 

Verified has no CI
Unveried has the standard ci+ a flag.